src/renderer/store/modules/launchFiles.ts

```diff
@@ -10,21 +10,21 @@ export interface LaunchFilesState {
 
 export const initialState: LaunchFilesState[] = [
```

Comment:
Editing the values for launch files directly in the UI would be a nice project. Just a thought, you can resolve this.

Reply:
Yep agreed, but we'll have to see how the launch handler will work once we'll start using Docker so I'll keep the project in mind for now